service/host_test.go

```diff
@@ -35,16 +35,16 @@ import (
 func TestOnCriticalError_NoObserver(t *testing.T) {
 	err := errors.New("Blargh")
 	sh := makeHost()
-	closeCh, ready, err := sh.Start(false)
-	require.NoError(t, err, "Expected no error starting up")
+	Control := sh.StartAsync()
```

Comment:
nit: local variables should never be capitalized, especially since you're shadowing a type here